storage/storage_bench.go

```diff
@@ -405,7 +405,7 @@ func AnnounceLeecher(b *testing.B, ps PeerStore) {
 // AnnounceLeecher1kInfohash can run in parallel.
 func AnnounceLeecher1kInfohash(b *testing.B, ps PeerStore) {
 	runBenchmark(b, ps, true, putPeers, func(i int, ps PeerStore, bd *benchData) error {
-		_, err := ps.AnnouncePeers(bd.infohashes[i%1000], false, 50, bd.peers[0])
```

Comment:
There's a reason this was `bd.infohashes[i%1000]`, namely that those are different infohashes, which are potentially stored in different shards of the storage, which allows this benchmark to run in parallel.